scripts/globals/items/flask_of_healing_powder.lua

```diff
@@ -0,0 +1,45 @@
+-----------------------------------------
+-- ID: 5322
+-- Item: Flask of Healing Powder
+-- Item Effect: Restores 25% HP AoE 10' radius
+-----------------------------------------
+require("scripts/globals/status")
+require("scripts/globals/msg")
+-----------------------------------------
+-- OnItemCheck
+-----------------------------------------
+
+function onItemCheck(target)
+    local result = 0
+    if target:getMaxHP() == target:getHP() then
+        result = 56 -- Does not let player use item if their hp is full
```

Comment:
wrong for this case - aoe item and its gonna work even if your entire party is full it'll just do nothing for anyone that was full. I notice you didn't do this on the item that does the same for MP so I'm guessing some copy paste form an existing item script happened.

Reply:
you are right sir lol

Reply:
Fixed